doc/developer-notes.md

```diff
@@ -461,6 +461,14 @@ namespace {
 
   - *Rationale*: Avoids confusion about the namespace context
 
+- Prefer `#include <primitives/transaction.h>` bracket syntax instead of
+  `#include "primitives/transactions.h" quote syntax when possible.
```

Comment:
Missing `.

Reply:
BTW, unrelated but the `Rationale` above doesn't render well. And some are missing a period.